website/migrations/0257_add_cve_indexes.py

```
@@ -0,0 +1,35 @@
+        migrations.AlterField(
+            model_name="issue",
+            name="cve_score",
+            field=models.DecimalField(blank=True, decimal_places=1, max_digits=3, null=True),
+        ),
+        # Increase CVE ID max_length to support longer CVE IDs
+        migrations.AlterField(
+            model_name="issue",
+            name="cve_id",
+            field=models.CharField(blank=True, max_length=20, null=True),
+        ),
+    ]
```

Ruff (0.14.6)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)